third_party/nvidia/lib/TritonNVIDIAGPUToLLVM/TritonGPUToLLVM.cpp

```diff
@@ -185,6 +186,8 @@ struct ConvertTritonGPUToLLVM
         id.replaceAllUsesWith(zero);
       });
     }
+    // Clear the linear layout cache after the conversion to LLVM is done.
+    triton::gpu::clearLinearLayoutCache();
```

Comment:
How about the AMD backend?

Reply:
yes, I have it in a local commit, forgot in the first commit